esphome/components/wifi/wifi_component.cpp

```diff
@@ -349,6 +349,9 @@ const LogString *get_signal_bars(int8_t rssi) {
 void WiFiComponent::print_connect_params_() {
   bssid_t bssid = wifi_bssid();
 
+  std::string m = get_mac_address();
+  ESP_LOGCONFIG(TAG, "  Local MAC: %c%c:%c%c:%c%c:%c%c:%c%c:%c%c", m[0], m[1], m[2], m[3], m[4], m[5], m[6], m[7], m[8],
+                m[9], m[10], m[11]);
```

Comment:
```suggestion
  ESP_LOGCONFIG(TAG, "  Local MAC: %s", get_mac_address_pretty().c_str());
```

Simpler and should to the same (well except the result being uppercase, but that shouldn't be a problem)

Reply:
Used your suggestion.